<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetHub</class>
 <widget class="QWidget" name="GetHub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>385</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GBWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Username </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="findButton">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>Repo Tree</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="infoLine">
         <property name="text">
          <string>Search for repositories by usernames</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
